packages/docs-ui/src/components/HeaderNavigation/HeaderNavigation.tsx

```diff
@@ -0,0 +1,43 @@
+import type React from 'react';
+import { Box, Hidden, HiddenVisually, Link, Text } from 'braid-design-system';
+import { MenuButton } from '../MenuButton/MenuButton';
+
+interface HeaderNavigationProps {
+  menuOpen?: boolean;
+  menuClick?: () => void;
+  logo: React.ReactNode;
+  logoLabel: string;
+  themeToggle: React.ReactNode;
```

Comment:
Thoughts on making this optional? I'll leave that up to you

Reply:
Actually sounds like the better idea